clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -252,6 +252,35 @@ public void testClientAuthenticationRequestedNotProvided() throws Exception {
 
         NetworkTestUtils.checkClientConnection(selector, node, 100, 10);
     }
+
+    /**
+     * Tests that an invalid SecureRandom implementation cannot be configured
+     */
+    @Test
+    public void testInvalidSecureRandomImplementation() throws Exception {
+        SslChannelBuilder channelBuilder = new SslChannelBuilder(Mode.CLIENT);
+        try {
+            sslClientConfigs.put(SslConfigs.SSL_SECURE_RANDOM_IMPLEMENTATION_CONFIG, "invalid");
+            channelBuilder.configure(sslClientConfigs);
+            fail("SSL channel configured with invalid SecureRandom implementation");
+        } catch (KafkaException e) {
+            // Expected exception
+        }
+    }
+
+    /**
+     * Tests that SecureRandom can be configured with SHA1PRNG
+     */
+    @Test
+    public void testSetSecureRandomSHA1PRNG() throws Exception {
+        SslChannelBuilder channelBuilder = new SslChannelBuilder(Mode.CLIENT);
+        try {
+            sslClientConfigs.put(SslConfigs.SSL_SECURE_RANDOM_IMPLEMENTATION_CONFIG, "SHA1PRNG");
```

Comment:
cc @rajinisivaram 
I'm not sure we should have this here - i.e., a unit test failure isn't particularly helpful if SHA1PRNG is unavailable (since the user may be fine with the null default).